### DISCUSSION TITLE
Save Image with Metadata question

### DISCUSSION THREAD

**Discussion by CanisDirusPrime** [crystian/ComfyUI-Crystools#43] [Q&A]
When I use the %date:mm-dd-yy% in the file name prefix field (i believe this is the "standard" way comfy creates new folders, but I'm fairly new to this so I'm probably missing something) to create and/or save an image to my outputs folder, all i get is an error.

![Screenshot 2024-03-25 152353](https://github.com/crystian/ComfyUI-Crystools/assets/160278975/a2ad30f6-473f-451a-82f3-00979c916910)

Any help would be greatly appreciated!

**Comment by crystian** [comment]
the template is not supported yet. Can you make a PR?
